samples/csharp_dotnetcore/06.using-cards/Dialogs/MainDialog.cs

```diff
@@ -21,49 +22,53 @@ public MainDialog(ILogger<MainDialog> logger)
         {
             _logger = logger;
 
-            AddDialog(new ChoicePrompt(nameof(ChoicePrompt)));
-            AddDialog(new WaterfallDialog(nameof(WaterfallDialog), new WaterfallStep[]
-            {
-                ChoiceCardStepAsync,
-                ShowCardStepAsync,
-            }));
+            // Define the main dialog and its related components.
+            AddDialog(new ChoicePrompt(nameof(ChoicePrompt)))
+                .AddDialog(new WaterfallDialog(nameof(WaterfallDialog), new WaterfallStep[]
+                {
+                    ChoiceCardStepAsync,
+                    ShowCardStepAsync,
+                }));
```

Comment:
seriously?

Reply:
The fluent chaining was for parity. With your recommendation I have unchained it in C# and Node.